modules/ROOT/generate-docs.sh

```diff
@@ -2,14 +2,42 @@
 
 set -exuo pipefail
 
-cd $(dirname $0)
```

Comment:
This allowed us to call the script from another directory, such as docs-site. Not a biggie, but a minor convenience while testing.

Reply:
I think it should still work? The script now cd's to `${GIT_ROOT}` straight away in order to do the sparse checkout: see line 8. It then cd's to `modules/ROOT` to generate the documents.

Reply:
Ah I understand. I'll put this line back in!

Reply:
This has its own downsides, but this script itself could cd to the directory of the script: 

```
SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
cd $SCRIPT_DIR
```

which would make it robust to running from out of directory.

Reply:
Done